fulp_modules/ipc/code/organs/stomach.dm

```diff
@@ -0,0 +1,22 @@
+/obj/item/organ/stomach/robot_ipc
+	name = "IPC micro cell"
+	icon = 'fulp_modules/ipc/icons/ipc_organs.dmi'
+	icon_state = "stomach-ipc"
+	w_class = WEIGHT_CLASS_NORMAL
+	zone = "chest"
+	slot = "stomach"
+	desc = "A specialised cell, for IPC use only. Do not swallow."
+	status = ORGAN_ROBOTIC
+	organ_flags = ORGAN_SYNTHETIC
+
+/obj/item/organ/stomach/robot_ipc/emp_act(severity)
+	. = ..()
+	if(!owner || . & EMP_PROTECT_SELF)
+		return
+	switch(severity)
+		if(1)
+			owner.nutrition = 50
+			to_chat(owner, "<span class='warning'>Alert: Detected severe battery discharge!</span>")
+		if(2)
+			owner.nutrition = 250
```

Comment:
does this mean that you can EMP yourself to gain nutrition if you have less than 250 nutrition left?